src/components/policy/test/counter_test.cc

```diff
@@ -31,7 +31,7 @@
 
 #include "gmock/gmock.h"
 
-#include "mock_statistics_manager.h"
+#include "include/test/policy/mock_statistics_manager.h"
```

Comment:
@OHerasym Please edit header path . Shoudl be used the following template : `<directory/file.h>`. Path to `directory` should be specified in CMakeList